CHANGELOG.md

```diff
@@ -1,47 +1,51 @@
 # Changelog
 
-## Unreleased
+## 4.5.0 (Feature release)
 
 ### New features
 
-#### Visually separate multiple summary lists on a single page with summary cards
+#### Use summary cards to visually separate multiple summary lists on a single page
 
-You can now use the [summary card](https://design-system.service.gov.uk/components/summary-list#summary-cards) to help you design and build pages with multiple summary lists to help you visually distinguish them and to allow users to use actions that apply to entire lists.
+You can now use the [summary card](https://design-system.service.gov.uk/components/summary-list#summary-cards).This new variant of the Summary list component can help you:
+
+design and build pages with multiple summary lists
```

Comment:
Do you want these as a sentence or do they need dashes? If as a sentence i need to amend the lead in line etc.

Reply:
This doesn't need a dash in front of it. We only need a dash when we either want to make a list or if we have 2 lines that we want to display with only a single line break in between eg lines 25 and 26, in which case markdown encourages us to use lists.